File: problems/algorithms/string-list-generator/js/response/list-generator.print.js
import { createList } from "./list-generator.js";

// List Generator

// making list
let expected = `<ul>
<li>Text 1</li>
<li>Text 2</li>
<li>Text 3</li>
</ul>
`;

console.log(createList(3));
console.log(expected);

// making list
expected = `<ul>
<li>Item 1</li>
<li>Item 2</li>
<li>Item 3</li>
<li>Item 4</li>
<li>Item 5</li>
</ul>
`;

console.log(createList(5, "Item"));
console.log(expected);

File: src/pages/exercises/string-list-generator/_codes/ruby/code/print/list_generator.print.rb
require_relative "../src/list_generator"

# List Generator

# making list
expected = "<ul>\n" +
                "  <li>Text 1</li>\n" +
                "  <li>Text 2</li>\n" +
                "  <li>Text 3</li>\n" +
            "</ul>"
puts ListGenerator.create_list(3)
puts expected


# making list
expected = "<ul>\n" +
                "  <li>Item 1</li>\n" +
                "  <li>Item 2</li>\n" +
                "  <li>Item 3</li>\n" +
                "  <li>Item 4</li>\n" +
                "  <li>Item 5</li>\n" +
            "</ul>"
puts ListGenerator.create_list(5, 'Item')
puts :equilateral


